### ISSUE TITLE
Clarify README

### ISSUE THREAD

**Issue by rpappalax** [mozilla/firefox-test-engineering#11]
The README is a good landing page for community contributors / new employees, but it's a bit unclear what is meant by:
```
Patches are always welcome! If you'd like to contribute, fork and make a pull request.
```
I believe this is referring to this very same mozilla/firefox-test-engineering docs repo, but perhaps we should give folks some indication here that there are test repos elsewhere they could jump into (and refer them to servicebook).

**Comment by stephendonner** [comment]
I agree this could potentially use a bit more clarity, but just to note the fuller context:

https://github.com/mozilla/firefox-test-engineering/blob/53127709c1e2bebaad6487319c92dfb009822015/README.rst appears to be visible in the GitHub repo view only, while https://github.com/mozilla/firefox-test-engineering/blob/53127709c1e2bebaad6487319c92dfb009822015/index.rst is the Read The Docs-default view, visible here: http://firefox-test-engineering.readthedocs.io/en/latest/index.html

In my mind, that begs the question: should we then update both documents, to point helpfully to Service Book (with some more context, and call-to-actions, of course)?

**Comment by rpappalax** [comment]
@stephendonner perhaps we should circle back again, once we've done our Q4 doc updates and talk thru the README to see if it makes sense or could use some updated word-smithing (including links to servicebook, etc.)
